README.md

```diff
@@ -51,49 +51,39 @@ Example: `./cleanup.sh ./stress.sh -c dfde16a004206cc92e21cc5a6cad9030fbe13c20 s
 
 #### Available tests
 ```
-note: This is subject to change
+Note: This is subject to change
 ```
 
 1. `cluster-test.json` : Creates an 8 node cluster and create 1000 collections of various `numShards` and measure shutdown & restart performance
 2. `stress-facets-local.json` : Indexes 20 million documents from an ecommerce events dataset, issues 5k facet queries against it.
+3. `prs-vs-nonprs.json` : Creates 1k collections using non-PRS mode (default), then restarts 7 nodes, then cleans up, and repeats the same for PRS.
 
 ### Results
 
-* Results are available after the benchmark in `./suites/results/results-\<configfile\>-\<commit-id\>.json` file. 
+* Results are available after the benchmark in `./suites/results/<testname (without the .json suffix)>/<commit-id>/results.json` file.
```

Comment:
Thanks for updating this! I made the change in stress.sh but didn't update README.md 😓